tests/src/Kernel/SmsFrameworkProviderTest.php

```diff
@@ -383,6 +388,7 @@ public function testEventsQueueIncoming() {
     $this->container->get('cron')->run();
 
     $expected[] = SmsEvents::MESSAGE_INCOMING_PRE_PROCESS;
+    $expected[] = 'incomingEvent'; // Plugin incoming event.
```

Comment:
Before this is committed I'm going to change the magic constant from __FUNCTION__ to __METHOD__ so it captures the full namespace and class name. So it is easier to reverse engineer where its coming from when looking at the testing/assertion code.

`$expected[] = 'incomingEvent';` will become `$expected[] = 'Drupal\sms_test_gateway\Plugin\SmsGateway\Memory::incomingEvent'`